frontend/src/app/applications/application-noncommercial-group/application-noncommercial-group.component.ts

```diff
@@ -1,19 +1,19 @@
-import { SpecialUseApplication } from '../../_models/special-use-application';
+import { alphanumericValidator } from '../validators/alphanumeric-validation';
 import { ApplicationService } from '../../_services/application.service';
 import { ApplicationFieldsService } from '../_services/application-fields.service';
 import { Component, OnInit } from '@angular/core';
 import { DateTimeRangeComponent } from '../fields/date-time-range.component';
 import { FormGroup, FormControl, FormArray, FormBuilder, Validators } from '@angular/forms';
 import { Router, ActivatedRoute } from '@angular/router';
+import { SpecialUseApplication } from '../../_models/special-use-application';
 import * as moment from 'moment/moment';
 
 @Component({
-  providers: [ ApplicationService, ApplicationFieldsService, DateTimeRangeComponent ],
+  providers: [ApplicationService, ApplicationFieldsService, DateTimeRangeComponent],
   selector: 'app-application-noncommercial-group',
   templateUrl: './application-noncommercial-group.component.html'
 })
 export class ApplicationNoncommercialGroupComponent implements OnInit {
-
   apiErrors: any;
   application = new SpecialUseApplication();
   forest = 'Mt. Baker-Snoqualmie National Forest';
```

Comment:
Something to think about going forward, is whether we want to make this dynamic now?